src/Providers.php

```diff
@@ -4,16 +4,20 @@
 
 namespace Interns2022B;
 
+use Illuminate\Support\Collection;
+use Interns2022B\Models\Brewery;
+
 class Providers
 {
     public const LIST = "list";
 
-    public function getProviders(array $data): array
+    public function getProviders(Collection $breweriesFactory): Collection
```

Comment:
I would like to rename breweriesFactory to breweries
Let's add doc above method name:
`/**  @var Collection<Brewery> $breweries * /`

Reply:
Renamed but composer deletes the doc